ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -57,69 +57,83 @@ StatusSectionLayout {
     signal backToCommunityClicked
     signal openLegacyPopupClicked // TODO: remove me when migration to new settings is done
 
-    leftPanel: ColumnLayout {
-        anchors {
-            fill: parent
-            margins: 8
-            topMargin: 16
-            bottomMargin: 16
-        }
-
-        spacing: 16
-
-        StatusNavigationPanelHeadline {
-            Layout.alignment: Qt.AlignHCenter
-            text: qsTr("Settings")
-        }
+    leftPanel: Item {
+        anchors.fill: parent
 
-        StatusListView {
-            id: listView
+        ColumnLayout {
+            anchors {
+                top: parent.top
+                bottom: footer.top
+                topMargin: 16
+                bottomMargin: 16
+                horizontalCenter: parent.horizontalCenter
+            }
+            width: parent.width - 16
+            spacing: 16
+            clip: true
 
-            Layout.fillWidth: true
-            implicitHeight: contentItem.childrenRect.height
+            StatusNavigationPanelHeadline {
+                Layout.alignment: Qt.AlignHCenter
+                text: qsTr("Settings")
+            }
 
-            model: root.settingsMenuModel
-            delegate: StatusNavigationListItem {
-                width: listView.width
-                title: modelData.name
-                asset.name: modelData.icon
-                selected: d.currentIndex === index
-                onClicked: d.currentIndex = index
+            StatusListView {
+                id: listView
+
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                model: root.settingsMenuModel
+                delegate: StatusNavigationListItem {
+                    width: listView.width
+                    title: modelData.name
+                    asset.name: modelData.icon
+                    selected: d.currentIndex === index
+                    onClicked: d.currentIndex = index
+                }
             }
         }
 
-        Item {
-            Layout.fillHeight: true
-        }
+        // TODO: remove me when migration to new settings is done. Only keep back button and anchor to it.
+        ColumnLayout {
+            id: footer
 
-        // TODO: remove me when migration to new settings is done
-        StatusBaseText {
-            Layout.alignment: Qt.AlignHCenter
-            text: qsTr("Open legacy popup (to be removed)")
-            color: Theme.palette.baseColor1
-            font.pixelSize: 10
-            font.underline: true
-
-            MouseArea {
-                anchors.fill: parent
-                cursorShape: Qt.PointingHandCursor
-                onClicked: root.openLegacyPopupClicked()
+            anchors {
```

Comment:
We have anchor only for bottom here, but we are setting `margins` and `topMargin` which are useless now.